homeassistant/components/comelit/const.py

```diff
@@ -0,0 +1,9 @@
+"""Comelit constants."""
+import logging
+
+_LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "comelit"
+
+DEFAULT_HOST = "192.168.1.252"
+DEFAULT_PIN = "111111"
```

Comment:
These are only used in the config flow, I'd suggest to move them there unless the rest of the implementation will need them too?

Reply:
I supposed that the const.py is the place for all constants; so to avoid duplicating them in future for oversight.
Anyway, done.